charts/astronomer/values.yaml

```
@@ -220,7 +220,17 @@ houston:
     # Default here is to run at 00:43 every night https://crontab.guru/#43_0_*_*_*
     schedule: "43 0 * * *"
 
+  # Populate daily task usage data
+  # This runs as a CronJob
+  populateDailyTaskMetrics:
+    # Default here is to run at 01:30 every night
+    schedule: "30 1 * * *"
```

It's best to avoid the hour of 1am for cron schedules due to DST caveats:

```
$ for X in {0..3} ; do gdate -d "2022-11-06T00:30:00-0400 +$X hours" "+%FT%T%z" ; done ;
2022-11-06T00:30:00-0400
2022-11-06T01:30:00-0400
2022-11-06T01:30:00-0500
2022-11-06T02:30:00-0500
```
```suggestion
  populateDailyTaskMetrics:
    # Default here is to run at 02:30 every night
    schedule: "30 2 * * *"
```

---

done